src/promptflow-evals/promptflow/evals/evaluators/chat/_chat.py

```diff
@@ -9,14 +9,15 @@
 
 import numpy as np
 
+from promptflow.core import AzureOpenAIModelConfiguration
 from promptflow.evals.evaluators import CoherenceEvaluator, FluencyEvaluator, GroundednessEvaluator, RelevanceEvaluator
 
 logger = logging.getLogger(__name__)
 
 
 class ChatEvaluator:
     def __init__(
-        self, model_config, eval_last_turn: bool = False, parallel: bool = True
+        self, model_config: AzureOpenAIModelConfiguration, eval_last_turn: bool = False, parallel: bool = True
```

Comment:
add PR description on why need to add this annotation

Reply:
I cherry-pick Han's commit https://github.com/microsoft/promptflow/pull/3065 to better test

Reply:
https://github.com/microsoft/promptflow/pull/3065 @YingChen1996 please remove these changes. such a PR will require review from evals team and mix it with msi change is confusing for them